src/org/usfirst/frc/team2335/robot/subsystems/Vision.java

```diff
@@ -16,7 +16,7 @@
 	//This is the distance we want to be from the target it feet
 	//TODO: Find appropriate distance from target
 	private final float wantedDistanceInFeet = 12.0f;
-	
+
     public float getDistance()
     {
     	//Use floats, they're way more accurate (AKA: doubles are the devil)
```

Comment:
This little section is just showing the changes made to the code, which is basically nothing.